src/internal/observable/from.ts

```diff
@@ -34,3 +34,74 @@ export function from<T>(input: ObservableInput<T>, scheduler?: SchedulerLike): O
 
   throw new TypeError((input !== null && typeof input || input) + ' is not observable');
 }
+
+/**
```

Comment:
Hi Rob, basically this is an awesome pr. Thanks for all the work you put into it. But this part has to be above the function declaration to be valid JSDoc. Maybe take a look at another operator like `fromEvent` then you will see what I mean.